src/applets/icon-tasklist/settings.ui

```diff
@@ -129,5 +129,29 @@
         <property name="top_attach">0</property>
       </packing>
     </child>
+    <child>
+      <object class="GtkLabel" id="label6">
+        <property name="visible">True</property>
+        <property name="can_focus">False</property>
+        <property name="halign">start</property>
+        <property name="label" translatable="yes">Middle click open new window</property>
```

Comment:
Should be: Middle Click To Launch New Window, similar to the summary in the dconf key.

Reply:
Respecting letter case of your suggestion goes against the other options labels case. Can we use `Middle click to launch new window`?

Reply:
Yep that'll work just fine

Reply:
Done